From 095cb8d965f6a88394623bf7bec3ffb2111788be Mon Sep 17 00:00:00 2001
From: Alpha Lin <alpha.lin@rock-chips.com>
Date: Tue, 9 Jun 2015 10:58:57 +0800
Subject: [PATCH] VPU: fix the code cannot be built in 32bits environment.

compat_uptr_t isn't define under 32bits environment, using
this definition under the CONFIG_COMPAT macro.

Signed-off-by: Alpha Lin <alpha.lin@rock-chips.com>
---
 arch/arm/mach-rockchip/vcodec_service.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-rockchip/vcodec_service.c b/arch/arm/mach-rockchip/vcodec_service.c
index b9f93242aa7f..967c3d158d3b 100755
--- a/arch/arm/mach-rockchip/vcodec_service.c
+++ b/arch/arm/mach-rockchip/vcodec_service.c
@@ -540,10 +540,12 @@ struct vpu_request {
 	u32 size;
 };
 
+#ifdef CONFIG_COMPAT
 struct compat_vpu_request {
 	compat_uptr_t req;
 	u32 size;
 };
+#endif
 
 /* debugfs root directory for all device (vpu, hevc).*/
 static struct dentry *parent;
-- 
2.35.3

